src/main/kotlin/no/nav/familie/ef/sak/fagsak/domain/Fagsak.kt

```diff
@@ -41,6 +42,7 @@ data class FagsakDomain(@Id
                         val eksternId: EksternFagsakId = EksternFagsakId(),
                         @Column("stonadstype")
                         val stønadstype: StønadType,
+                        val sistGOmregnet: LocalDate? = null,
```

Comment:
Når vi gjør oppretter en førstegangsbehandling for en person, skal vi då sette sistGOmregnet? 
Jeg lurer på om det hadde vært fint å vi burde vurdere å sette denne på tilkjent ytelse i stedet?

Då kan man sette den fra beregning, som oppretter en ny tilkjent ytelse

Så lurer jeg då på om man skulle bruke en string som versjon, eks "2022", som man kan eks sette til "2022...<annet>" hvis man har behov for det